config/testdata/load_config

```diff
@@ -0,0 +1,19 @@
+[default]
+region = ap-south-1
+
+[profile default]
+region = ap-north-1
+
+[profile duplicate-profile]
+region = us-east-1
+
+[profile duplicate-profile]
+region = us-west-2
```

Comment:
If not already present, would be helpful to unit test that cover validating that sections that should be dropped are correctly done so.

```ini
# Config file
[default]
[profile default]

[profile foo]
[foo]
```
```ini
# credentials file
[profile foo]
[foo]
```